docs/docusaurus/docs/cloud/connect/connect_airflow.md

```diff
@@ -58,49 +59,54 @@ Apache Airflow is an orchestration tool that allows you to schedule and monitor
     ```python title="Python"
     import os
     import great_expectations as gx
-    from airflow import DAG
-    from airflow.operators.python import PythonOperator
-    from datetime import datetime
-
-    # Replace <YOUR_ACCESS_TOKEN> and <YOUR_CLOUD_ORGANIZATION_ID> with your credentials.
-
-    GX_CLOUD_ACCESS_TOKEN = "<YOUR_ACCESS_TOKEN>"
-    GX_CLOUD_ORGANIZATION_ID = "<YOUR_CLOUD_ORGANIZATION_ID>"
+    import pendulum
+    from airflow.decorators import dag, task
 
-    CHECKPOINT_NAME = "<YOUR_CHECKPOINT>"
 
-    def run_gx_airflow():
-
-        context = gx.get_context()
-        checkpoint = context.get_legacy_checkpoint(name=CHECKPOINT_NAME)
-        checkpoint.run()
-
-    default_args = {
-        'owner': 'airflow',
-        'depends_on_past': False,
-        'start_date': datetime(2023, 8, 9),
-    }
-
-    gx_dag = DAG(
-        'gx_dag',
-        default_args=default_args,
-        schedule_interval= '0 0 * * *', # This is set to run daily at midnight. Adjust as needed.
-        catchup=False
+    @dag(
+        schedule=None,
+        start_date=pendulum.datetime(2023, 8, 9),
+        catchup=False,
     )
-
-    run_gx_task = PythonOperator(
-        task_id='gx_airflow',
-        python_callable=run_gx_airflow,
-        dag=gx_dag,
-    )
-
-    run_gx_task
+    def gx_dag_with_deco():
+        os.environ["NO_PROXY"] = "*" #https://github.com/apache/airflow/discussions/24463
+        print("Great Expectations DAG Started")
+
+        @task
+        def run_checkpoint():
+            print("Running Checkpoint")
+            # Replace <YOUR_ACCESS_TOKEN>, <YOUR_CLOUD_ORGANIZATION_ID>, and <CHECKPOINT_NAME> with your credentials
+            GX_CLOUD_ACCESS_TOKEN = ""
+            GX_CLOUD_ORGANIZATION_ID = ""
+            CHECKPOINT_NAME = ""
```

Comment:
`non-blocking`
```suggestion
            # alternativey set CHECKPOINT_NAME to be a runtime parameter
            CHECKPOINT_NAME = ""
```
I don't think we need to complicate the example by using an airflow parameter, but it's probably worth mentioning they can do it this way.
https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/params.html#task-level-params

Reply:
@nicgrayson did you intend to make this change?
If you did it looks like it was overwritten.